### ISSUE TITLE
Unable to manage domain records

### ISSUE THREAD

**Issue by wassafr** [Gandi/gandi.cli#296]
I have 17 domains, all are LiveDNS (as confirmed in dashboard, table view) and managed by admin v5 (some of theme were migrated from v4). I can't create any record, either by either v4 ou v5 API. I've triple checked all API keys, even regenerated the v5 key multiple times.

```
$ gandi domain list
<14 domains out of 17>

$ gandi dns domain.list
<empty result>

$ gandi record create --name toto --type CNAME --value titi plipplop.com
No zone records found, domain plipplop.com doesn't seems to be managed at Gandi.

$ gandi dns create plipplop.com toto CNAME titi
Sorry domain plipplop.com does not exist
Please use one of the following:
```

Of course plipplop.com is one of the 14 domains listed by `gandi domain list`

So I don't understand why:
* The `dns` command doesn't return any result while `domain` does although all my domains are LiveDNS
* Why only 14 out of 17 domains are returned?
* Why `gandi record create` says plipplop.com doesn't exist

Thanks for your help

**Comment by wassafr** [comment]
Oh boy this is so confusing. I think I've found out why it's not working.

When I migrated from v4 to v5, Gandi created 2 "organizations" : one is a "personal organization" and is basically my username, the other one is the name of my company.

All services and billing information are linked to the 2nd and it seems that, although my account is the owner of the organization, it doesn't get the domains when using the CLI.

I managed to get the domains by calling manually the LiveDNS API and providing the sharing_id of my company organization but I couldn't find such an option for the CLI.

**Comment by wassafr** [comment]
Oh boy this is so confusing. I think I've found out why it's not working.

When I migrated from v4 to v5, Gandi created 2 "organizations" : one is a "personal organization" and is basically my username, the other one is the name of my company.

All services and billing information are linked to the 2nd and it seems that, although my account is the owner of the organization, it doesn't get the domains when using the CLI.

I managed to get the domains by calling manually the LiveDNS API and providing the sharing_id of my company organization but I couldn't find such an option for the CLI.
